scripts/config.py

```diff
@@ -493,9 +494,13 @@ def _get_compatible_GCC(self, nvcc_version):
         elif 9.2 <= nvcc_version < 10.1:
             return v[v.index('7.3'):]
         elif 10.1 <= nvcc_version <= 10.2:
-            return v[v.index('8.4'):]
-        elif 11.0 <= nvcc_version <= 11.2:
+            return v[v.index('8.5'):]
+        elif 11.0 <= nvcc_version <= 11.4:
+            # nvcc 11.4.0 --> gcc 9.3
+            # nvcc 11.4.1 --> gcc 11
             return v[v.index('9.3'):]
+        elif 11.5 <= nvcc_version <= 11.6:
+            return v[v.index('9.3'):]#gcc 11 is suported by nvcc not by XMIPP
         return []  # not supported
```

Comment:
not on this line, but we should report that user has unsupported CUDA version, instead of reporting that no compatible GCC compiler has been found